pkgs/applications/editors/jetbrains/versions.json

```diff
@@ -3,157 +3,157 @@
       "url-template": "https://download.jetbrains.com/go/goland-{version}.tar.gz",
-      "version": "2022.1.3",
-      "sha256": "b96a1cb1b6be1d1e55c85de551fbd94b9a8c0566193d784f3bc408da60e6904e",
-      "url": "https://download.jetbrains.com/go/goland-2022.1.3.tar.gz",
+      "version": "2022.2.1",
+      "sha256": "6c9ba39a96d802121908b136435220f99206c7e2e874eccee9ab0a75ec305762",
+      "url": "https://download.jetbrains.com/go/goland-2022.2.1.tar.gz",
       "version-major-minor": "2022.1",
-      "build_number": "221.5921.26"
+      "build_number": "222.3345.146"
     },
     "idea-community": {
       "update-channel": "IntelliJ IDEA RELEASE",
       "url-template": "https://download.jetbrains.com/idea/ideaIC-{version}.tar.gz",
-      "version": "2022.1.3",
-      "sha256": "f8a14e3ab100cf745dc7e329e13bd31961cd728c6b7676493b9ffb4e290a9543",
-      "url": "https://download.jetbrains.com/idea/ideaIC-2022.1.3.tar.gz",
+      "version": "2022.2",
+      "sha256": "bbec46c56ae7c6fe92f2a16af0e3bd6a4c50786198535d368030ee24e520b997",
+      "url": "https://download.jetbrains.com/idea/ideaIC-2022.2.tar.gz",
       "version-major-minor": "2022.1",
-      "build_number": "221.5921.22"
+      "build_number": "222.3345.118"
     },
     "idea-ultimate": {
       "update-channel": "IntelliJ IDEA RELEASE",
       "url-template": "https://download.jetbrains.com/idea/ideaIU-{version}-no-jbr.tar.gz",
-      "version": "2022.1.3",
-      "sha256": "97748cd2dacf74701eb4ec30227d96a9b5ab5c09afc7e59443669ab839d20d02",
-      "url": "https://download.jetbrains.com/idea/ideaIU-2022.1.3-no-jbr.tar.gz",
+      "version": "2022.2",
+      "sha256": "dcecd027c7f723fb615ad411884670c759e2a7428c3010f3d789e99b10536fe3",
+      "url": "https://download.jetbrains.com/idea/ideaIU-2022.2-no-jbr.tar.gz",
       "version-major-minor": "2022.1",
-      "build_number": "221.5921.22"
+      "build_number": "222.3345.118"
     },
     "mps": {
       "update-channel": "MPS RELEASE",
       "url-template": "https://download.jetbrains.com/mps/{versionMajorMinor}/MPS-{version}.tar.gz",
       "version": "2021.3.1",
       "sha256": "b7d41c4362e71f30adeaed9f0ec30afd5ac0e6eea9650ee4a19d70a5783db3e6",
-      "url": "https://download.jetbrains.com/mps/2021.3/MPS-2021.3.1.tar.gz",
+      "url": "https://download.jetbrains.com/mps/2021.3.1/MPS-2021.3.1.tar.gz",
       "version-major-minor": "2021.3",
       "build_number": "213.7172.958"
     },
     "phpstorm": {
       "update-channel": "PhpStorm RELEASE",
       "url-template": "https://download.jetbrains.com/webide/PhpStorm-{version}.tar.gz",
-      "version": "2022.1.3",
-      "sha256": "ea5d005f3aee2588dd725bd3aa87d5e3beb51ac87b1505f34263f9da3babd360",
-      "url": "https://download.jetbrains.com/webide/PhpStorm-2022.1.3.tar.gz",
+      "version": "2022.2",
+      "sha256": "91c96d078c26018e7994372b4eef1c2c76772473e13c1dded06ff10de31495f4",
+      "url": "https://download.jetbrains.com/webide/PhpStorm-2022.2.tar.gz",
```

Comment:
PHPStorm 2022.2.1 has been released, please use that version.

Reply:
Just updated them all

Reply:
Cool, thanks !